### DISCUSSION TITLE
docker服务端是否会生成文件或数据

### DISCUSSION THREAD

**Discussion by zuohaoda** [Jeric-X/SyncClipboard#80] [General]
感谢有这么好的软件
1、问下docker部署的服务端，是否会生成上传的文件或复制的内容（历史）
2、独立部署服务器和WebDAV挂载的区别是什么？WebDAV能保存文件和内容，独立部署服务器还有什么优势吗？

**Comment by Jeric-X** [comment]
1、没有对外展示历史记录的功能，但是历史文件是存储在docker容器内的/app/file/目录里，可以使用docker命令行查看，客户端有一个“自动删除服务器上的临时文件”的设置会影响到这个目录
2、独立部署就是给用户多一个选择，如果已经有了一个运行中的WebDAV服务，独立部署服务器是没有任何优势的。但是在未来独立部署的服务器可能会有更多的功能，比如历史记录汇总，服务端的历史记录控制、文件大小限制等等

**Comment by zuohaoda** [comment]
谢谢，期待后面的功能
- 服务端的历史记录控制
- 历史记录汇总
